cg/meta/orders/schema.py

```diff
@@ -110,22 +110,22 @@ def validate(self, value):
 
     # This information is required
     'name': validators.RegexValidator(NAME_PATTERN),
-    'container': validators.Any(CONTAINER_OPTIONS),
+    'container': validators.Optional(TypeValidator(str, allow_none=True), None),
     'data_analysis': str,
     'application': str,
     'sex': validators.Any(SEX_OPTIONS),
     'family_name': validators.RegexValidator(NAME_PATTERN),
     'require_qcok': bool,
     'tumour': bool,
-    'source': str,
+    'source': validators.Optional(TypeValidator(str, allow_none=True), None),
```

Comment:
Should prolly move to the "Order portal specific" block on line 109

Reply:
I think I will update the comments..